StudyBuddy/Managers/FileManager.cs

```diff
@@ -1,32 +1,31 @@
 ﻿using System.Globalization;
+using CsvHelper;
+using CsvHelper.Configuration;
```

Comment:
If you import whole of CsvHelper is the CsvHelper.Configuration needed?

Reply:
IDE no like if no CsvHelper.Configuration